manifests/config.pp

```diff
@@ -34,5 +34,15 @@
     mode   => '0755',
   }
 
+  file { $katello::repo_export_dir:
+    ensure  => directory,
+    owner   => $katello::user,
+    group   => $katello::group,
+    mode    => '0755',
+    # the default export dir is /var/lib/pulp/katello-export, this 'require'
+    # ensures that the parent dirs exist without having to define the full
+    # tree.
+    require => Package['pulp-server'],
```

Comment:
It shouldn't? I don't see why it would error for a requires, it should just install the pkg if its not there.